recipe/meta.yaml

```diff
@@ -24,14 +24,14 @@ requirements:
         - python
         - setuptools
         - numpy
-        - cdat-lite
-        - udunits2
+        - netCDF4
+        - cfunits
```

Comment:
It looks like the entrypoint changed :unamused:

Let me take a look at this...